app/webapp/src/main/java/uk/ac/ebi/fg/annotare2/web/gwt/editor/client/view/experiment/design/AssignFilesDialog.java

```diff
@@ -87,19 +87,24 @@ public AssignFilesDialog(DialogCallback<Map.Entry<FileType, Integer>> callback,
             @Override
             public void onChange(ChangeEvent event) {
                 columnListHelp.setHTML(values.get(columnListBox.getSelectedIndex()).getHelpText());
-                if(experimentProfileType.isMicroarray() && values.get(columnListBox.getSelectedIndex()).isProcessed()){
-                    noOfColumnsLabel.setVisible(true);
-                    noOfColumns.setVisible(true);
+                if(experimentProfileType.isSequencing() || (experimentProfileType.isMicroarray() && values.get(columnListBox.getSelectedIndex()).isProcessed())){
```

Comment:
Could you please extract these conditional checks to a private method and give a well descriptive name to it?
Thanks